chia/util/initial-config.yaml

```diff
@@ -633,6 +633,8 @@ wallet:
   # The amount someone has to pay you in mojos for you to see their notification
   required_notification_amount: 10000000
 
+  use_delta_sync: False
```

Comment:
I believe there's a risk of missing updated when enabling this. for example, if there's a reorg while syncing, I believe we may miss updates by failing to re-play/re-request the rolled-back heights.

If the wallet is synced to a higher block than all the puzzle hash subscription, I also believe we may not even request all the updates.

Anyway, if this works in some (and perhaps most) situations, I'm fine with adding it. Perhaps there should be a bit of an explanation and warning label here though, in the config file

Reply:
> If the wallet is synced to a higher block than all the puzzle hash subscription, I also believe we may not even request all the updates.

Can you elaborate more on this?

Reply:
[This new CHIP](https://github.com/Chia-Network/chips/pull/100) should also make it easier to ensure that reorgs don't mess up coin state.